recipes/cairo/meson/conanfile.py

```diff
@@ -113,7 +115,7 @@ def requirements(self):
             self.requires("egl/system")
 
     def build_requirements(self):
-        self.build_requires("meson/0.59.1")
+        self.build_requires("meson/0.59.3")
```

Comment:
```suggestion
        self.build_requires("meson/0.60.2")
```

Any reason not to jump to `0.60`?

Reply:
no, it's [bincrafters-conventions bot](https://github.com/bincrafters/bincrafters-conventions) which commited this change, and he doesn't know yet about meson 0.60